### DISCUSSION TITLE
DSA problems

### DISCUSSION THREAD

**Discussion by AmitDatir** [AbhishekKudal/java-practice#5] [Q&A] [answered]
Can you please suggest a good youtube channel for DSA problems?

**Comment by AbhishekKudal** [comment, marked as answer]
You can refer the DSA bootcamp by Kunal Kushwaha [here](https://www.youtube.com/watch?v=rZ41y93P2Qo&list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ&ab_channel=KunalKushwaha)
